app/ht_indexer/src/document_generator/document_generator_service.py

```diff
@@ -85,7 +85,7 @@ def log_error_document_generator_service(self, e, document, delivery_tag):
                                                                      e, document)
 
         logger.error(f"Document {document.get('ht_id')} failed {error_info}")
-        self.src_queue_consumer.reject_message(self.src_queue_consumer.conn.ht_channel,
+        self.src_queue_consumer.reject_message(self.src_queue_consumer.ht_channel,
```

Comment:
These changes to expose methods on `src_queue_consumer` rather than reaching through to the underlying connection seem like a good refactoring